src/Microsoft.Net.Http.Headers/project.json

```diff
@@ -24,6 +24,7 @@
   "frameworks": {
     "netstandard1.1": {
       "dependencies": {
+        "Microsoft.Extensions.Primitives": "1.1.0-*",
```

Comment:
But does it hurt to move it under `netstandard`? Is this what you're recommending library authors do?